### PR TITLE
Documentation on how to customize default 404 response for unmapped requests

### PR DESCRIPTION
**Pull request by mvogiatzis**
Fixes issue: https://github.com/tomakehurst/wiremock/issues/507

### CODE REVIEW

**Comment by mvogiatzis** [comment]
Is this the right place for documenting this ?

**Comment by tomakehurst** [comment]
I think this probably belongs in the stubbing section rather than simulating faults.

Also, there's now an easier way to match all URLs:

```java
  stubFor(any(anyUrl()))
```

**Comment by mvogiatzis** [comment]
Sounds good, I will move it and improve code.

**Comment by tomakehurst** [comment]
Thanks

File: docs-v2/_docs/stubbing.md [hunk 1]
```diff
@@ -265,6 +265,41 @@ JSON documents):
...
+                        .withBody("{\"status\":\"Error\",\"message\":\"Endpoint not found\"}")));
+```
+
+In JSON
+
+```json
+{
+  "priority":10,
+  "request": {
+    "method": "GET",
```
Comment (tomakehurst) [inline docs-v2/_docs/stubbing.md]: Agree `ANY` would probably be better

**Pull request merged**